testing/sliding-sync-integration-test/src/lib.rs

```diff
@@ -734,6 +734,95 @@ mod tests {
         Ok(())
     }
 
+    #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+    async fn selective_unfreeze_with_add() -> anyhow::Result<()> {
+        let (_client, sync_proxy_builder) = random_setup_with_rooms(10).await?;
+        print!("setup took its time");
+        let build_views = |growing_cached: bool| {
+            let sliding_window_view = SlidingSyncViewBuilder::default()
+                .sync_mode(SlidingSyncMode::Selective)
+                .set_range(1u32, 10u32)
+                .sort(vec!["by_recency".to_string(), "by_name".to_string()])
+                .name("sliding")
+                .build()?;
+            let growing_sync = SlidingSyncViewBuilder::default()
```

Comment:
```suggestion
            let growing_sync_view = SlidingSyncViewBuilder::default()
```

?

Reply:
it is in the `build_views` fn, but if you insist, I don't care.